src/main/java/com/bandwidth/sqs/consumer/SqsConsumer.java

```diff
@@ -154,19 +156,27 @@ public int getNumPermits() {
      * Shuts down the consumer, and blocks until the shutdown completes with a
      * default timeout of 30 seconds
      */
-    public void shutdown() {
-        shutdown(DEFAULT_SHUTDOWN_TIMEOUT);
+    @PreDestroy
+    public boolean shutdown() {
+        return shutdown(DEFAULT_SHUTDOWN_TIMEOUT);
     }
 
     /**
      * Shuts down the consumer, and blocks until the shutdown completes.
      * Throws a timeout exception if it fails to shutdown in the timeout specified
      */
-    public void shutdown(Duration timeout) {
+    public boolean shutdown(Duration timeout) {
+        LOG.info("Beginning graceful shutdown of SQS Consumer");
         permitChangeDisposable.dispose();
-        shutdownAsync()
-                .timeout(timeout.getSeconds(), TimeUnit.SECONDS)
-                .blockingAwait();
+        boolean completed = shutdownAsync()
+                .blockingAwait(timeout.getSeconds(), TimeUnit.SECONDS);
+
+        if(completed) {
+            LOG.info("Graceful shutdown of SQS consumer complete");
```

Comment:
Unsure if we want this as info, since we're trying to cut back on log ingest to sumo

Reply:
This would only get logged once when the app shuts down, not per transaction or anything

Reply:
This provides audiability and traceability for graceful shutdowns.  It may be logged 10's of times a day, not millions.